yarn-project/aztec.js/src/contract/sent_tx.ts

```diff
@@ -15,15 +15,15 @@ export type WaitOpts = {
    * If false, then any queries that depend on state set by this transaction may return stale data. Defaults to true.
    **/
   waitForNotesSync?: boolean;
-  /** Whether newly created notes should be included in the receipt. */
-  getNotes?: boolean;
+  /** Whether to include information useful for testing in the receipt. */
+  test?: boolean;
```

Comment:
Can you say a little on why this was called test and not debug? I am not sure what I would expect if passing along a test flag, but debug, debugInfo or includeDebugInfo seems like something that could be more telling.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/3221/commits/0b51791a21a775fe7675220b1f2213bc4ef07e19